OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -118,6 +118,9 @@ public Order IssueOrder(Actor self, IOrderTargeter order, in Target target, bool
 
 		public void ResolveOrder(Actor self, Order order)
 		{
+			if (order.OrderString == "Stop")
+				Path.Clear();
```

Comment:
```suggestion
			{
				Path.Clear();
				return;
			}
```

You can include a return statement here for even better readability and incredibly insignificant optimisation 😄

Reply:
Every cpu cycle counts :-)